setup.py

```diff
@@ -26,7 +26,8 @@
     'protobuf',
     'schema',
     'solrq',
-    'validators'
+    'validators',
+    'keyring'
```

Comment:
Have you considered making this a platform specific dependency? `MacOS == Darwin` See https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies

Reply:
Yes, I completely forgot about this, thank you. I will add a comment for the future in case we want to support other platforms and tools such as Linux or other specific backends since that keyring supports a lot of credential managers.

Reply:
You initial attempt was only a partial implementation. See this PR https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/225 on how it could be performed. However let's raise this in stand up tomorrow to discuss adding support for more than macOS as the keyring package supports other OS keyrings